### ISSUE TITLE
Customizable cursor activation area

### ISSUE THREAD

**Issue by clinta** [feschber/lan-mouse#182]
Currently an entire side of the monitor activates the next client with no consideration for differing monitor sizes.

I have a much larger monitor on my server, below a smaller client monitor. When I move the cursor to the top of the screen, even if that area is not below the physical client monitor, the cursor moves up to the client. It would be nice if I could tell lanmouse a more precise activation area, like top [x]:[y], where [x] and [y] are min and max pixel positions. So only movement up within that area activates the client.

I think this would also open up the possibility of having multiple clients on one side. For example two top clients, one of the left side, and one on the right.

**Comment by feschber** [comment]
Yeah this is something I will get to eventually. I'm also thinking it would be nice to make it configurable interactively
